Tests/UITests/TestApp/TestAppDelegate.swift

```diff
@@ -6,16 +6,17 @@
 // SPDX-License-Identifier: MIT
 //
 
+import Foundation
 import Spezi
 import SpeziScheduler
 
 
-typealias TestAppScheduler = Scheduler<TestAppStandard, String>
+typealias TestAppScheduler = Scheduler<String>
 
 
 class TestAppDelegate: SpeziAppDelegate {
     override var configuration: Configuration {
-        Configuration(standard: TestAppStandard()) {
+        Configuration(standard: ExampleStandard()) {
```

Comment:
You should be able to completely go without a `Standard` here as we don't have any requirements on the standard and can just inject a default instance that is used within Spezi.
```suggestion
        Configuration {
```